spec/ics-007-tendermint-client/README.md

```diff
@@ -84,6 +84,7 @@ The Tendermint client tracks the timestamp (block time), validator set, and comm
 ```typescript
 interface ConsensusState {
   timestamp: uint64
+  processedTime: uint64
```

Comment:
ConsensusState should not contain `processedTime`

Reply:
Aye, updated.